plugin.video.nhkworldtv/lib/first_run_wizard.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from kodi_six import xbmc, xbmcgui
+
+
+def show_wizard(ADDON):
+    """Shows the first run wizard
+    """
+
+    if (not ADDON.getSettingBool('run_wizard')):
+        return (False)
+
+    xbmc.log("Starting Set-up Wizard")
+    dialog = xbmcgui.Dialog()
+
+    delimiter = '\n\n'
+    lines = []
+    lines.append('NHK World TV comes with a set of configurable options.')
+    lines.append('You can always change them in the addon-settings later.')
```

Comment:
Sorry, strings shown in the UI have to be localized, please move them to strings.po